data/json/items/book/chemistry.json

```diff
@@ -373,6 +575,72 @@
     "category": "manuals",
     "name": { "str": "Biodiesel: Renewable Fuel Resource", "str_pl": "copies of Biodiesel: Renewable Fuel Resource" },
     "description": "A large textbook for college students about biodiesel.",
+    "variants": [
+      {
+        "id": "textbook_biodiesel",
+        "name": { "str": "Biodiesel: Renewable Fuel Resource", "str_pl": "copies of Biodiesel: Renewable Fuel Resource" },
+        "description": "A large textbook for college students about biodiesel.",
+        "weight": 1
+      },
+      {
+        "id": "textbook_biodiesel_1",
+        "name": {
+          "str": "Biodiesel Production: Technologies, Challenges, and Future Prospects",
+          "str_pl": "copies of Biodiesel Production: Technologies, Challenges, and Future Prospects"
+        },
+        "description": "Sponcored by American Society of Civil Engineers, this book presents approaches, technologies, and source materials for biodiesel production, as well as socioeconomic and political impacts of his applications.  You surely don't need the second part anymore.",
+        "weight": 1
+      },
+      {
+        "id": "textbook_biodiesel_2",
+        "name": { "str": "The Biodiesel Handbook", "str_pl": "copies of The Biodiesel Handbook" },
+        "description": "Big, 500-page book, that provide plenty info about history of different fuels, methods to apply them, and required properties for them, mainly center on biodiesel and another biofuels.",
+        "intelligence": 8,
+        "weight": 1
+      },
+      {
+        "id": "textbook_biodiesel_3",
+        "name": {
+          "str": "Biodiesel: a Realistic Fuel Alternative for Diesel Engines",
+          "str_pl": "copies of Biodiesel: a Realistic Fuel Alternative for Diesel Engines"
+        },
+        "description": "Small book, that offer insights into plant-based fuels, with detailed analysis of biodiesel, and possible scenarios how it can be implemented in .",
```

Comment:
```suggestion
        "description": "A small book offering insights into plant-based fuels, with detailed analysis of biodiesel, and possible scenarios for their implementation.",
```

Reply:
Huh, seems i forgot to finish this one :derpy: